governance/votes/Executive vote - June 13, 2024.md

```diff
@@ -0,0 +1,169 @@
+summary: Disable the Starknet Dai Bridge, distribute Aligned Delegate compensation for May 2024, distribute AVC Member compensation for Q2 2024, transfer funding to the Launch Project, trigger a Spark Proxy spell, and perform housekeeping actions on the USDP Jar.
+date: 2024-06-13T00:00:00.000Z
+address: "$spell_address"		
+---
+
+# [Executive Proposal] Disable Starknet Dai Bridge, Aligned Delegate Compensation, AVC Member Compensation, Launch Project Funding, Spark Proxy Spell, USDP Jar Housekeeping - June 13, 2024
+		
+The Governance Facilitators and Sidestream, Dewiz, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+		
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+		
+---
+		
+## Executive Summary
+		
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- The Starknet Dai Bridge will be disabled.
+- Aligned Delegate compensation for May 2024 totaling **210.81 MKR** will be distributed.
+- Aligned Voter Committee Member compensation for Q2 2024 totalling **100.08 MKR** will be distributed.
+- **5,000,000 DAI** and **450 MKR** will be transferred to the Launch Project.
+- A Spark proxy spell will be triggered at $address.
+- A series of housekeeping actions to push USDP out of the input conduit will be carried out, as detailed below.
+		
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+		
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **30 hours**.
+		
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. 
+		
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+		
+		
+---
+		
+## Proposal Details
+		
+### Disable Starknet Dai Bridge
+		
+- **Authorization:** [Forum Post](https://forum.makerdao.com/t/starknet-dai-handover/22033/13)
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/starknet-dai-handover/22033/12)
+		
+As part of the transition from the old Starknet Dai Bridge to a new version maintained by Starkgate, deposits on the old bridge will be disabled, if this executive proposal passes. Withdrawals will still be possible. To do this the following action needs to occur:
```

Comment:
I think it would be easier to read this sentence changing the order of its elements to: "If this executive proposal passes, deposits on the old Starknet Dai Bridge will be disabled as part of the transition to a new bridge version maintained by Starkgate".

Reply:
I think it is fine in either format.